cmatrix.c

```diff
@@ -32,8 +32,6 @@
 #include <termios.h>
 #include <signal.h>
 
-#include "config.h"
```

Comment:
config.h is generated using make and is needed for some configuration options that we will leave for the user.